### DISCUSSION TITLE
Common Access Card (CAC) Support?

### DISCUSSION THREAD

**Discussion by usmcamp0811** [qutebrowser/qutebrowser#7551] [Q&A] [answered]
I just made the switch to `qutebrowser` and am curious if there is any way to be able to use a CAC with the browser. I am able to set it up following the [Arch Wiki on CAC](https://wiki.archlinux.org/title/Common_Access_Card) and make Brave work with it.. but when I try to use it in `qutebrowser` its a no go. Any idea? Is this even possible?

**Comment by The-Compiler** [comment, marked as answer]
This is the first time I hear of this. If I'm reading the wiki page right, it needs some kind of master password entered when you use it? If so, the missing bit in QtWebEngine is probably the same as this: [[QTBUG-90938] Support FIDO2 user verification - Qt Bug Tracker](https://bugreports.qt.io/browse/QTBUG-90938)

**Comment by usmcamp0811** [comment]
yea when you goto a CAC enabled site it will prompt for a PIN; so yea this does look to be the same thing. Thanks
